### PR TITLE
Implement Delete Participant Command

### PR DESCRIPTION
**Pull request by ChocoRagdoll**
changes made

The DeleteCommand file and its residue should be removed by someone after this is merged. I can do it on Wednesday.

### CODE REVIEW

**Review by ChocoRagdoll — COMMENTED** [review]
no s i wrote this myself

File: src/main/java/seedu/address/logic/commands/DeletePersonCommand.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        }
+
+        List<Person> lastShownList = model.getFilteredPersonListOfSelectedEvent();
+        if (targetIndex.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        }
+
+        Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
+        if (!model.isPersonInSelectedEvent(personToDelete)) {
+            throw new CommandException(MESSAGE_NO_SUCH_PERSON);
```
Comment (Apzure) [inline src/main/java/seedu/address/logic/commands/DeletePersonCommand.java:100]: Test this condition

File: src/main/java/seedu/address/logic/commands/DeletePersonCommand.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        if (!model.isAnEventSelected()) {
+            return deleteFromGlobal(model);
+        } else {
+            return deleteFromEvent(model);
+        }
```
Comment (Apzure) [inline src/main/java/seedu/address/logic/commands/DeletePersonCommand.java:54]: Make the condition `(model.isAnEventSelected())` instead for readability, then reverse the statements accordingly

**Review by Apzure — CHANGES_REQUESTED** [review]
LGTM, just a few minor changes

**Review by myang2020 — APPROVED** [review]

**Review by Apzure — APPROVED** [review]
LGTM!

**Pull request merged**